assertk-common/src/main/kotlin/assertk/table.kt

```diff
@@ -6,7 +6,7 @@ private class TableFailure(private val table: Table) : Failure {
     private val failures: MutableMap<Int, MutableList<AssertionError>> = LinkedHashMap()
 
     override fun fail(error: AssertionError) {
-        failures.getOrPut(table.index, { ArrayList() }) += error
+        failures.getOrPut(table.index, { ArrayList() }).plusAssign(error)
```

Comment:
This change is needed because he `+=` seems to not work in javascript

Reply:
That seems odd. Do you know if there is a reason for this or is it a bug?

Reply:
I didn't dived strong enough into it :/ when using the `+=` I have a compilation exception. Look like a bug at compile time but not sure :/